### PR TITLE
Add support for framework distribution via Swift Package Manager.

### PR DESCRIPTION
**Pull request by auhustsinovich**
## What were done

- Created `Package.swift` file:
   - Explicitly specified the `static` type of library.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.10
```
Comment (AlexLevshenkov) [inline Package.swift:1]: nit: 
does it allow to add local package via Xcode 15.2 (it uses swift 5.9)? Or have you succeeded with Xcode 15.3?

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+let package = Package(
+    name: "Collections",
+    platforms: [
+        .macOS(.v10_13), .iOS(.v12),
+    ],
+    products: [
+        .library(
+            name: "Collections",
+            type: .static,
+            targets: ["Collections"]),
```
Comment (AlexLevshenkov) [inline Package.swift:15]:
```suggestion
        .library(
            name: "Collections",
            type: .static,
            targets: ["Collections"]
            ),
```

File: Package.swift (hunk 1 above)
Comment (auhustsinovich) [reply Package.swift:1]: @AlexLevshenkov 
I was able to accomplish this in the following way, in Xcode 15.3:
1. Downloaded the necessary branch as a zip archive, unzipped it
2. Added this archive to my project
3. Changed the path to the framework, from absolute from `Downloads` to relational in the root folder of my project
Done:)

File: Package.swift (hunk 2 above)
Comment (auhustsinovich) [reply Package.swift:15]: fixed,thanks!

**Review by AlexLevshenkov — APPROVED** [review]

**Pull request merged**